deps.windows/70-libdatachannel.ps1

```diff
@@ -0,0 +1,84 @@
+param(
+    [string] $Name = 'libdatachannel',
+    [string] $Version = '0.18.5',
+    [string] $Uri = 'https://github.com/paullouisageneau/libdatachannel.git',
+    [string] $Hash = '709a66339451bb4c8d4e5ced78c67605ec09da31',
+    [array] $Targets = @('x64'),
+    [switch] $ForceShared = $true
+)
+
+function Setup {
+    Invoke-GitCheckout -Uri $Uri -Commit $Hash
+}
+
+function Clean {
+    Set-Location $Path
+
+    if ( Test-Path "build_${Target}" ) {
+        Log-Information "Clean build directory (${Target})"
+        Remove-Item -Path "build_${Target}" -Recurse -Force
+    }
+}
+
+function Configure {
+    Log-Information "Configure (${Target})"
+    Set-Location $Path
+
+   if ( $ForceShared -and ( $script:Shared -eq $false ) ) {
+        $Shared = $true
```

Comment:
Do we need to check if `$script:Shared` is false here? @PatTheMav this does seem to mimic existing patterns, but what do you think?

Reply:
i ripped that again from Pat's ps1 scripts in https://github.com/obsproject/obs-deps/pull/186

Reply:
since there's no word from Pat, and since it's his code, i'll resolve for now.